packages/core/src/hydration/api.ts

```diff
@@ -143,6 +142,21 @@ function whenStableWithTimeout(appRef: ApplicationRef, injector: Injector): Prom
   return whenStablePromise;
 }
 
+/**
+ * Defines a name of an attribute that is added to the <body> tag
+ * in the `index.html` file in case a given route was configured
+ * with `RenderMode.Client`. 'cm' is an abbreviation for "Client Mode".
+ */
+export const CLIENT_RENDER_MODE_FLAG = 'ngcm';
+
+/**
+ * Checks whether the `RenderMode.Client` was defined for the current route.
+ */
+function isClientRenderModeEnabled() {
```

Comment:
NIT: prefer using an explicit return type.

```suggestion
function isClientRenderModeEnabled(): boolean {
```